### PR TITLE
Bio-Formats 5.5.0

### PR DESCRIPTION
**Pull request by sbesson**
As discussed with @jburel @hflynn and @lunson, this PR updates Bio-Formats pages to match the latest Bio-Formats 5.5.0

### CODE REVIEW

**Comment by hflynn** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/products/bio-formats/downloads/ built on staging

File: products/bio-formats/downloads/index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ---     
         <div class="callout large primary" id="bg-image-bio-formats">
             <div class="row column text-center">
-                <h1>Bio-Formats [5.x.x] Downloads</h1>
+                <h1>Bio-Formats 5.5.0 Downloads</h1>
                 <p>[ a blurb about the Downloads ]</p>
```
Comment (hflynn) [inline products/bio-formats/downloads/index.html]: We'll need to remove this or add some text here too "Download to latest version here"?

Comment (sbesson) [inline products/bio-formats/downloads/index.html]: Yes I wasn't sure about that to put there. There is no similar template in OMERO.

Comment (lunson) [inline products/bio-formats/downloads/index.html]: @hflynn @sbesson We can always delete it unless we can put some useful note there.

Comment (sbesson) [inline products/bio-formats/downloads/index.html]: `Download the latest version` does not bring much information over the header I would say. I propose to delete for now.

Comment (hflynn) [inline products/bio-formats/downloads/index.html]: We can just delete it if you don't think having extra text there is informative/needed

**Comment by hflynn** [comment]
What's the plan for previous versions? Add a 'Previous' tab and then link back to http://downloads.openmicroscopy.org/bio-formats/ ?

**Comment by lunson** [comment]
@hflynn That works for me.

**Comment by hflynn** [comment]
This looks fine on staging, do we want to merge it for now or were you going to add the previous tab yourself @sbesson ?

**Comment by sbesson** [comment]
Assuming we are fine with this as an intermediate step, I'd capture the previous tab as a follow-up on the reference card (alongside OMERO and the other exposed software)

**Comment by hflynn** [comment]
👍

**Pull request merged**